zxlive/commands.py

```diff
@@ -12,6 +12,8 @@
 from pyzx.graph import GraphDiff
 from pyzx.utils import EdgeType, VertexType
 
+from zxlive.w_node import get_w_partner, is_w_edge_type, is_w_vertex_type
```

Comment:
Shouldn't this be `from .w_node import ...`?

Reply:
both works. this was probably auto-imported by VS-code. I will change it to `from .w_node` for consistency.